src/CommandingOfficers/CommanderAve.java

```diff
@@ -432,10 +442,8 @@ protected void perform(MapMaster gameMap)
           if((tileEnvi.weatherType != Weathers.SNOW) || (tileEnvi.terrainType == TerrainType.FOREST))
           {
             // Destroy any forests. Big hail, man.
-            TerrainType newTerrain = (loc.getEnvironment().terrainType == TerrainType.FOREST) ? TerrainType.GRASS : loc.getEnvironment().terrainType;
```

Comment:
You definitely still need this conditional
![image](https://user-images.githubusercontent.com/8674241/53224803-9575b380-363b-11e9-8ec6-36805cbb2590.png)

Reply:
It might also be good to redraw the terrain around changed squares, so that transitions can be updated.
![image](https://user-images.githubusercontent.com/8674241/53224952-0f0da180-363c-11e9-88e7-b54eea2d9291.png)

Reply:
Hah! That's what I get for pushing stuff when I was supposed to be asleep. Fantastic.

And good thought on transitions. For weather changes, it isn't necessary, but for other things, yeah.